### PR TITLE
NOISSUE: add const for initialBalance in tests

### PR DESCRIPTION
**Pull request by ElSacramento**
**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: ledger-core/v2/application/functest/api_test_utils.go [hunk 1]
```diff
@@ -150,8 +150,8 @@ func getWalletBalance(url, ref string) (int, error) {
 }
 
 // Adds amount to wallet.
-func addAmountToWallet(url, ref string, amount uint) error {
-	rawResp, err := sendAPIRequest(url, walletAddAmountRequestBody{To: ref, Amount: amount})
+func addAmountToWallet(url, ref string, amount int) error {
```
Comment (ivanshibitov) [inline ledger-core/v2/application/functest/api_test_utils.go:153]: why did you change uint to int?

Comment (ElSacramento) [reply ledger-core/v2/application/functest/api_test_utils.go:153]: To do typecast in one place
